extensions/amp-nested-menu/0.1/test/test-amp-nested-menu.js

```diff
@@ -38,7 +38,7 @@ describes.realWin(
       clock = lolex.install({target: win});
 
       // TODO(#25022): remove this toggle when cleaning up experiment post launch.
```

Comment:
The issue number here should also be updated to https://github.com/ampproject/amphtml/issues/25343.